### ISSUE TITLE
Segmentation fault with calling external procedure pointer

### ISSUE THREAD

**Issue by shingo-s** [omni-compiler/omni-compiler#386]
`F_Front` raises segmentation fault with the following code.

```fortran
      SUBROUTINE s(f)
        EXTERNAL :: f
        POINTER :: f
        PRINT *, f()
      END SUBROUTINE s
```

**Comment by shingo-s** [comment]
https://github.com/omni-compiler/omni-compiler/pull/473 will fix this issue

**Comment by shingo-s** [comment]
This issue is already fixed in f30d4d2df911e12a745b8f942d9b29449da70e58
